app/routes/post.tsx

```
@@ -0,0 +1,73 @@
+                  year: 'numeric',
+                  month: 'short',
+                  day: 'numeric',
+                })}
+              </time>
+              <span>{post.readTime} min read</span>
+            </div>
+          </header>
+
+          <div
+            className="prose-enhanced max-w-none"
+            dangerouslySetInnerHTML={{ __html: htmlContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)